src/sage/quadratic_forms/bqf_class_group.py

```diff
@@ -611,3 +643,178 @@ def order(self):
             2
         """
         return order_from_multiple(self, self.parent().cardinality())
+
+
+def _project_bqf(bqf, q):
+    r"""
+    Internal helper function to compute the image of a
+    :class:`BQFClassGroup_element` of discriminant `D`
+    in the form class group of discriminant `D/q^2`.
+
+    EXAMPLES::
+
+        sage: from sage.quadratic_forms.bqf_class_group import _project_bqf
+        sage: f1 = BinaryQF([4, 2, 105])
+        sage: f2 = _project_bqf(f1, 2); f2
+        x^2 + x*y + 105*y^2
+        sage: f1.discriminant().factor()
+        -1 * 2^2 * 419
+        sage: f2.discriminant().factor()
+        -1 * 419
+
+    ::
+
+        sage: f1 = BinaryQF([109, 92, 113])
+        sage: f2 = _project_bqf(f1, 101); f2
+        53*x^2 - 152*x*y + 109*y^2
+        sage: f1.discriminant().factor()
+        -1 * 2^2 * 101^2
+        sage: f2.discriminant().factor()
+        -1 * 2^2
+
+    ALGORITHM: Find a class representative with `q^2 \mid a`
+    (and `q\mid b`) and substitute `x\mapsto x/q`.
```

Comment:
```suggestion
    ALGORITHM:

    Find a class representative with `q^2 \divides a`
    (and `q \divides b`) and substitute `x\mapsto x/q`.
```
For usual latex `\divides` is better (operator and clearly differentiates between a set separator), but I am not sure for mathjax.

Can we also move this before the `EXAMPLES::`?

Reply:
According to a quick test, `\divides` seems undefined in MathJax.

Reply:
Thanks. That's what I was thinking, but I appreciate you checking and confirming.